lib/Map/LeafletPatched.js

```diff
@@ -0,0 +1,27 @@
+import L from "leaflet";
+
+// Function taken from Leaflet 1.0.1 (https://github.com/Leaflet/Leaflet/blob/v1.0.1/src/layer/vector/Canvas.js#L254-L267)
+// Leaflet 1.0.2 and later don't trigger click events for every Path, so feature selection only gives 1 result.
+// Updated to incorporate function changes up to v1.7.1
+L.Canvas.prototype._onClick = function(e) {
+  var point = this._map.mouseEventToLayerPoint(e),
+    layers = [],
+    layer;
+  for (var order = this._drawFirst; order; order = order.next) {
+    layer = order.layer;
+    if (layer.options.interactive && layer._containsPoint(point)) {
+      if (
+        !(e.type === "click" || e.type !== "preclick") ||
+        !this._map._draggableMoved(layer)
+      ) {
+        L.DomEvent.fakeStop(e);
+        layers.push(layer);
+      }
+    }
+  }
+  if (layers.length) {
+    this._fireEvent(layers, e);
+  }
+};
+
+export default L;
```

Comment:
Thanks for porting this!

Reply:
I'm currently getting this when testing locally with the "GeoJSON Test" catalog item

![image](https://user-images.githubusercontent.com/6187649/179664724-c8515de0-af7c-4d58-a5ea-eba93309310b.png)

Reply:
Initially, when I implemented this we were on 1.7 of Leaflet, in 1.8 fakeStop is removed and apparently not needed anymore as it is removed in the original 1.8 implementation
https://github.com/Leaflet/Leaflet/pull/7439/files#diff-f4127fe2c74bbf5c584e01246849fac1954ead8521449585b61d268bfb7b059eL358